processor/metricsgenerationprocessor/processor_test.go

```diff
@@ -391,76 +391,101 @@ func getOutputForIntGaugeTest() pmetric.Metrics {
 	return intGaugeOutputMetrics
 }
 
-func TestSumCalculateNewMetric(t *testing.T) {
-	next := new(consumertest.MetricsSink)
-	cfg := &Config{
```

Comment:
This configuration is now done by the file `testdata/input_metric_types/config.yaml`.